server/tests/secrets/test_secret_ref_utils.py

```
@@ -0,0 +1,320 @@
+    )
+    def test_canonical_providers_present(self, must_have):
+        """Anchor the spelling of headline providers; rename = visible CI break."""
+        assert must_have in SUPPORTED_SECRET_PROVIDERS
+
+
+# ---------------------------------------------------------------------------
+# Case-insensitive provider lookup
+# ---------------------------------------------------------------------------
+
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)